react-components/select/step3.md

```diff
@@ -0,0 +1,47 @@
+With a basic select component in place, build a more complex example in this step, by using the available variants.
+
+The select component has four variants: single, checkbox, typeahead, and multiple option typeahead. When no variant is specified, the component defaults to the single variant. The single and typeahead variants allow for a single selection at a time, while the checkbox and multiple option typeahead variants allow for multiple selections at a time.
+
+1) **Add `SelectVariant` to the list of imports from @patternfly/react-core.** With this import in place, this enum's options may be used rather than a string literal. This enum looks like the following:
```

Comment:
They don't need to add anything for the enum beyond the import, it's one step. I'll remove the structure display, since the tutorial tells them what to use in the next step.